airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/form_submissions.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": ["null", "object"],
+    "properties": {
+      "submittedAt": {
+        "type": ["null", "string"],
+        "format": "date-time"
+      },
+      "values": {
+        "type": ["null", "array"],
+        "items": {
+          "type": ["null", "object"],
+          "properties":{
+            "name": {
+              "type": ["null", "string"]
+            },
+            "value": {
+              "type": ["null", "string"]
+            },
+            "objectTypeId": {
+              "value": {
+                "type": ["null", "string"]
+              }
+            }
+          }
+        }
+      },
+      "pageUrl": {
+        "type": ["null", "string"]
+      },
+      "form_guid":{
+        "type": ["null", "string"]
+      }
+    }
+  }
```

Comment:
please, add empty string in the end of file

Reply:
added